plugins/scale/scale.cpp

```diff
@@ -787,6 +787,43 @@ class wayfire_scale : public wf::plugin_interface_t
             target_alpha);
     }
 
+    static bool view_compareX(const wayfire_view& A, const wayfire_view& B)
+    {
+        auto vgA = A->get_wm_geometry();
```

Comment:
We try to steer away from camelCase but instead use snake_case. Here I would probably use ...`vg_a = view_a->`...

Reply:
done